astropy/modeling/tests/test_quantities_parameters.py

```diff
@@ -295,9 +295,9 @@ def test_parameter_quantity_comparison():
     assert g.mean == 1 * u.m
     assert 1 * u.m == g.mean
     assert g.mean != 1
-    assert 1 != g.mean
+    assert 1 != g.mean  # noqa: SIM300
 
-    assert g.mean < 2 * u.m
+    assert 2 > g.mean  # noqa: SIM300
```

Comment:
```suggestion
    assert g.mean < 2 * u.m # noqa: SIM300
```
Should fix the failure

Reply:
Oh. Thanks!

Reply:
Missed the `* u.m`